applications/cdh_breakout_board_fc_sim/tasks/inter_subsystem_rs485.py

```diff
@@ -7,13 +7,17 @@
 import microcontroller
 import busio
 import digitalio
-
+import pin_manager
 
 async def inter_subsystem_rs485_sender_task():
     """
     Task that sends 0xFFEEDDCC and lights up the LED for 1 second if write is successful.
     """
-    led = digitalio.DigitalInOut(board.LED1)
+    
+    #led = digitalio.DigitalInOut(board.LED1) # OLD
+    led = pin_manager.PinManager.get_instance()
+    led.create_digital_in_out(board.LED1) # should be all that's needed for context managing
```

Comment:
> should be all that's needed for context managing

No? If you read the pin_manager module documentation you'll see an example which includes what you need for context managing in good Python code, the `with` statement:

> ```py
> with m_dio as gpio:
>     gpio.direction = digitalio.Direction.OUTPUT
>     gpio.value = True